src/main/java/picard/illumina/parser/IlluminaFileUtil.java

```diff
@@ -265,6 +265,11 @@ public static boolean hasCbcls(final File basecallDir, final int lane) {
         final File laneDir = new File(basecallDir, IlluminaFileUtil.longLaneStr(lane));
         final File[] cycleDirs = IOUtil.getFilesMatchingRegexp(laneDir, IlluminaFileUtil.CYCLE_SUBDIRECTORY_PATTERN);
 
+        // Either the lane or the cycle directory do not exist!
+        if (cycleDirs == null) {
+            return false;
```

Comment:
As this is an error, not an empty directory...should we throw/catch a more detailed error, or at least log something here?